analysis/report_measures.py

```diff
@@ -0,0 +1,75 @@
+from ehrql import INTERVAL, create_measures, months, codelist_from_csv
+from ehrql.tables.tpp import clinical_events, patients, practice_registrations
```

Comment:
`patients` table is not used in this definition, but we might want to use variables like age or sex from this later?

Reply:
Yep, we should be using it later for sex and age.